app/helpers/react_components/mentoring/representations/representation.rb

```diff
@@ -9,10 +9,23 @@ def to_s
             representation: SerializeExerciseRepresentation.(representation),
             examples: examples_data,
             mentor: mentor_data,
+            mentor_solution:,
             guidance: {
-              representations_html: Markdown::Parse.(track.mentoring_representations).presence,
-              track_mentoring_notes_html: track.mentoring_notes_content.presence,
-              exercise_mentoring_notes_html: exercise.mentoring_notes_content.presence
+              representationsHtml: Markdown::Parse.(track.mentoring_representations).presence,
```

Comment:
This shouldn't be camel-cased, it should be underscored. It's down to JS to do that conversion (and there's a function we use in the bootstrap file for it). 

But I also don't think it needs the `html` suffix at all any more.

```suggestion
              representations: Markdown::Parse.(track.mentoring_representations).presence,
```

Reply:
oh, true! yes, I readded the html suffix, because it sounded clearer what this is for. but I will remove it.

Reply:
I quite like the `_html` suffix, and we use it in several other places too. Not having it here makes it feel somewhat inconsistent to me.